pikaur_test/test_errors.py

```diff
@@ -29,6 +29,32 @@ def test_install_not_found(self):
             not_existing_pkg_name
         )
 
+    def test_install_not_found_ignore(self):
```

Comment:
i can't understand from these two testcases what's the real life usecase scenario for this flag - could you please elaborate more on it?

Reply:
If you have a list of packages that you want to install, it does not break when packages do not exist (anymore)

e.g. `pikaur -S linux notexisting` breaks, `pikaur -S linux notexisting --ignore-missing-aur` installs `linux` and ignores the not existing `notexisting`

The other one is just a flag that you can set in config

Reply:
and why it need to be speicifc for aur, not just "ignore-missing-packages" ?

Reply:
i don't see any practical usecase for ignoring only aur or only repo packages missing - so i suggest one common flag to just ignore any missing packages

Reply:
ok, i changed it to the general case and added the check before the other exit(6)